src/common/utils.cpp

```diff
@@ -40,6 +40,21 @@ double time_now(void) {
     return result;
 }
 
+void utc_time_now(struct tm **utc) {
+    time_t tv;
+    time(&tv);
+    *utc = gmtime(&tv);
```

Comment:
Could you refactor this as

```C
*utc = gmtime(time(nullptr));
```

Rationale for this request is the following excerpt of text from the time(3) on Linux:

```man
BUGS
       Error returns from this system call are indistinguishable from
       successful reports that the time is a few seconds before the Epoch, so
       the C library wrapper function never sets errno as a result of this
       call.

       The tloc argument is obsolescent and should always be NULL in new code.
       When tloc is NULL, the call cannot fail.
```